net-misc/openntpd/openntpd-6.2_p3-r1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd user
+
+NTP_HOME="${NTP_HOME:=/var/lib/openntpd/chroot}"
+MY_P="${P/_p/p}"
+
+DESCRIPTION="Lightweight NTP server ported from OpenBSD"
+HOMEPAGE="http://www.openntpd.org/"
+SRC_URI="mirror://openbsd/OpenNTPD/${MY_P}.tar.gz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd"
+IUSE="libressl selinux"
+
+DEPEND="
+	!<=net-misc/ntp-4.2.0-r2
+	!net-misc/ntp[-openntpd]
+	libressl? ( dev-libs/libressl:0= )"
+
+RDEPEND="
+	${DEPEND}
+	selinux? ( sec-policy/selinux-ntp )"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	enewgroup openntpd 321
```

Comment:
You should use the GLEP-81, aka "acct" method of handling groups and users. Check how it's done with **ntp** package for example. You'll also need to send a mail to gentoo-dev mailing list requesting UID+GID for your acct-* packages.